### ISSUE TITLE
GitHub Error: remote: no anonymous write access

### ISSUE THREAD

**Issue by coco-nebula** [microsoft/vscode#137161]
Issue Type: <b>Bug</b>

Since yesterday pushing to git does not work anymore. I tried using access tokens, initialize a new repository, everything. On other editors it works just fine, i think it's a vscode issue.

VS Code version: Code 1.62.2 (3a6960b964327f0e3882ce18fcebd07ed191b316, 2021-11-11T20:51:30.084Z)
OS version: Linux x64 5.11.0-38-generic snap
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 5700U with Radeon Graphics (16 x 2383)|
|GPU Status|2d_canvas: unavailable_software<br>gpu_compositing: disabled_software<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: disabled_off<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: unavailable_software<br>webgl2: unavailable_software|
|Load (avg)|1, 1, 1|
|Memory (System)|13.53GB (6.59GB free)|
|Process Argv|--no-sandbox . --crash-reporter-id ffe1ec00-3103-4563-92e4-fa859f103db5|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|ubuntu|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|ubuntu|
|XDG_SESSION_TYPE|x11|
</details><details><summary>Extensions (11)</summary>

Extension|Author (truncated)|Version
---|---|---
bracket-pair-colorizer|Coe|1.0.61
es7-react-js-snippets|dsz|3.1.1
gitlens|eam|11.6.1
prettier-vscode|esb|9.0.0
vscode-styled-components|jpo|1.6.6
python|ms-|2021.11.1422169775
vscode-pylance|ms-|2021.11.1
emoji|Per|1.0.1
material-icon-theme|PKi|4.10.0
LiveServer|rit|5.6.1
quokka-vscode|Wal|1.0.418

(2 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30283811
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30384385
pythondataviewer:30285071
vscod805cf:30301675
pythonvspyt200:30340761
binariesv615:30325510
vsccppwtct:30382698
bridge0708:30335490
dockerwalkthru:30377721
bridge0723:30353136
pythonrunftest32:30373476
pythonf5test824:30373475
javagetstartedt:30391933
pythonvspyt187:30373474
vsqsis400:30386382
vsaa593cf:30376535
vssld246cf:30386378

```

</details>

**Comment by gjsjohnmurray** [comment]
Maybe #137063

Assuming you are on 1.62.2, does a downgrade to 1.62 work around the issue until a fix arrives?

**Comment by faresdjefaflia** [comment]
I have the same problem

**Comment by gjsjohnmurray** [comment]
> I have the same problem

Does a downgrade work as a workaround?

**Comment by musasutisna** [comment]
I have the same problem.
I work with a bunch of workspaces.
I open a workspace with a terminal still opened that will load the last saved session on I type env will show variable TERM_PROGRAM_VERSION=1.62.1 and git push is working
on another workspace without terminal opened, I open the terminal and type env will show variable TERM_PROGRAM_VERSION=1.62.2 with git push is not working

**Comment by amanv8060** [comment]
@gjsjohnmurray Based on above response , I believe it is a duplicate of #137063

**Comment by faresdjefaflia** [comment]
> > I have the same problem
> 
> Does a downgrade work as a workaround?

I haven't tried and I don't know how to go back

**Comment by gjsjohnmurray** [comment]
> I haven't tried and I don't know how to go back

Guidance in  #136791. Basically, first turn off auto-update, then download and install 1.62.0 from the link that suits your platform (which you didn't mention).

For even earlier versions, start at https://code.visualstudio.com/updates/ and pick the date on the left. Then use the download links from the top of its release notes.

**Comment by gjsjohnmurray** [comment]
Or go to https://github.com/microsoft/vscode/releases

**Comment by faresdjefaflia** [comment]
> > I haven't tried and I don't know how to go back
> 
> Guidance in #136791. Basically, first turn off auto-update, then download and install 1.62.0 from the link that suits your platform (which you didn't mention).
> 
> For even earlier versions, start at https://code.visualstudio.com/updates/ and pick the date on the left. Then use the download links from the top of its release notes.

Now I have reverted to the old version and now everything is fine and all the problems have been resolved. It seems that the problem is in the latest version and I hope that it will be fixed. I am using Linux, an Ubuntu distribution.
https://code.visualstudio.com/updates/v1_61

**Comment by lmiq** [comment]
Same problem here (on 1.62). I downgraded to 1.61.2 and the problem went away. (On Linux Mint 20.1)

**Comment by gjsjohnmurray** [comment]
/duplicate of #137183 which is scheduled to be fixed in upcoming recovery release (1.62.3)

**Comment by HackMEAny** [comment]
> /duplicate of #137183 which is scheduled to be fixed in upcoming recovery release (1.62.3)

Yes it worked for me reverted back using arch aur(git revert)

**Comment by ppatierno** [comment]
@gjsjohnmurray I have 1.62.3 and still the same issue:

```
remote: No anonymous write access.
```

This is my env:

```
Version: 1.62.3
Commit: ccbaa2d27e38e5afa3e5c21c1c7bef4657064247
Date: 2021-11-17T08:00:36.721Z
Electron: 13.5.2
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Linux x64 5.14.18-100.fc33.x86_64
```
